contracts/staking/StakingInfo.sol

```diff
@@ -257,6 +258,7 @@ contract StakingInfo {
         view
         returns (
             uint256 amount,
+            uint256 reward,
```

Comment:
If you are adding a return value, then I'd expect this to be used somewhere but I don't see an equivalent change in this PR?

Reply:
oh on Heimdall? no, that pr is fixing these changes.
Since stakingInfo is helper contract this function is meant for things like staking-API and other dapps interacting with it.